src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 7.3
 ---
 
+ * `mailer.message_logger_listener` now registered in DI only when profiler enabled to prevent memory leaks
```

Comment:
shouldn't be needed for a bugfix

Reply:
I'm not sure whether it is a bugfix

Reply:
... because solution is about to dropping service from DI  container when profiler is disabled

Reply:
updated to classify these changes as a bugfix - dropped changelog entry

Thank you